docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
+
+The specification can be roughly grouped into the following categories:
+
+* Parameters that describe the training process: **trainer**
+* Parameters that describe the datasets: **model.dataset**, **model.train_ds**, **model.validation_ds**
+* Parameters that describe the model: **model**
+
+More details about parameters in the config file could be found below and in the `model's config file <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml>`__:
+
+
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| **Parameter**                             | **Data Type**   |  **Description**                                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| pretrained_model                          | string          | Path to the pre-trained model .nemo file or pre-trained model name                                           |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.dataset.data_dir                    | string          | Path to the data converted to the specified above format                                                     |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| trainer.max_epochs                        | integer         | Maximum number of epochs to train the model                                                                  |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.tokenizer.tokenizer_name            | string          | Tokenizer name, will be filled automatically based on model.language_model.pretrained_model_name             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.tokenizer.vocab_file                | string          | Path to tokenizer vocabulary                                                                                 |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.tokenizer.tokenizer_model           | string          | Path to tokenizer model (only for sentencepiece tokenizer)                                                   |
```

Comment:
let's have a page containing the language_model section as it is common among all nlp models. Then just give a reference here.